Formula/px.rb

```diff
@@ -0,0 +1,35 @@
+class Px < Formula
+  include Language::Python::Virtualenv
+
+  desc "Ps and top for human beings (px / ptop)"
+  homepage "https://github.com/walles/px"
+  url "https://github.com/walles/px.git",
+    using:    :git,
+    tag:      "1.4.0",
+    revision: "aea41c2a67a8b83e644e11d8ac152e2c123d0ee1"
```

Comment:
```suggestion
      tag:      "1.4.0",
      revision: "aea41c2a67a8b83e644e11d8ac152e2c123d0ee1"
```
We don't need `using: :git`. Also, let's align the arguments to `url`, please.